### PR TITLE
Create sarahva47's first post with a trinket and markdown formatting

### PR DESCRIPTION
**Pull request by sarahva47**

### CODE REVIEW

File: _posts/sarahva47/2021-10-20-sarahva47-trinket-post.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Here's an example of how to use Regex (regular expressions) to make a list. I chose this example because I was pretty confused due to the weird syntax of emails, and couldn't wholly figure it out until we discussed this problem in class, so I'm taking a moment to document my understanding of this problem for future regex use.:
+
+<iframe src="https://trinket.io/embed/python/f945d6ce15" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+This problem from chapter 12 takes a file, pulls out all email addresses, and puts them in a list of emails.
+
+This piece of code sets up the problem by first importing the information needed to use regex, then getting the file, and defining the list. This is important to do first so that everything is set up. :
+
+...
```
Comment (IMBLAH) [inline _posts/sarahva47/2021-10-20-sarahva47-trinket-post.md]: I think this should be ```.

**Review by IMBLAH — COMMENTED** [review]
I think this should be ```.

File: _posts/sarahva47/2021-10-20-sarahva47-trinket-post.md (hunk 1 above)
Comment (sarahva47) [reply _posts/sarahva47/2021-10-20-sarahva47-trinket-post.md]: thank you :)

**Review by MishaCienfuegos — APPROVED** [review]
This formatting looks spot on!

**Pull request merged**